gradle.properties

```
@@ -3,8 +3,8 @@ org.gradle.parallel=true
 fabric.loom.multiProjectOptimisation=true
 
 archivesBaseName=quilted_fabric_api
-version=7.0.6
-upstream_version=0.85.0
+version=7.0.7
```

keep it 7.0.6 for now, it's a PR;
```suggestion
version=7.0.6
```
it's not going to be 7.0.7 anyway